Tests/SentryTests/Integrations/OutOfMemory/SentryOutOfMemoryTrackerTests.swift

```diff
@@ -207,7 +207,27 @@ class SentryOutOfMemoryTrackerTests: NotificationCenterTestCase {
         sut.start()
         assertNoOOMSent()
     }
-    
+
+    func testAppOOM_WithBreadcrumbs() {
+        let breadcrumb = Breadcrumb()
+        breadcrumb.level = SentryLevel.info
+        breadcrumb.timestamp = Date(timeIntervalSince1970: 10)
+        breadcrumb.category = "category"
+        breadcrumb.type = "user"
+        breadcrumb.message = "Click something"
+
+        let sentryOutOfMemoryScopeObserver = SentryOutOfMemoryScopeObserver(maxBreadcrumbs: 10, fileManager: fixture.fileManager)
+        sentryOutOfMemoryScopeObserver.addSerializedBreadcrumb(breadcrumb.serialize())
+
+        sut.start()
+        goToForeground()
+
+        fixture.fileManager.moveAppStateToPreviousAppState()
+        fixture.fileManager.moveBreadcrumbsToPreviousBreadcrumbs()
+        sut.start()
+        assertOOMEventSent(expectedBreadcrumbs: 1)
```

Comment:
`m`: I think it would be better to check if the added breadcrumb matched the one above, instead of only checking if there is one breadcrumb.

Reply:
Isn't that already covered by the `testStoreBreadcrumb` test?

Reply:
Okay, fine by me.